### PR TITLE
Add post for sprint 91 about notifications

### PR DESCRIPTION
**Pull request by saraycp**

### CODE REVIEW

**Comment by hennevogel** [comment]
Deploy preview for *openbuildservice* ready!

Built with commit 60b0a64ba8ef11614c851e0a59a90bd445c65088

https://deploy-preview-282--openbuildservice.netlify.app

File: _posts/development/2021-02-12-notifications-new-filters.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+Those of you enrolled in the [beta program](https://openbuildservice.org/2018/10/04/the-beta-program)
+already know the new Notifications page we introduced a while ago.
+After some interations refining its functionality and UI, we have also introduced two more filters:
+_Incoming Requests_ and _Outgoing Requests_ filters.
+Keep reading to know more about them, you could find them really useful.
+
+## Filters Overview
+
+During a normal day, you can receive a big amount of web notifications. This, of course, depends on how many projects and
+packages you are involved in. The notifications' filters are there to help you find what you are looking for.
```
Comment (danidoni) [inline _posts/development/2021-02-12-notifications-new-filters.md:16]:
```suggestion
During a normal day, you may receive a big amount of web notifications. The notifications' filters are there to help you find what you are looking for.
```

File: _posts/development/2021-02-12-notifications-new-filters.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+[screenshot of filters]
+
+
+In previous iterations we already introduced different notifications' filters, check it out in the following posts:
+
+[link to previous blog posts where other filters were introduced]
+
+Attending to the users' feedback, we have introduced a new pair of filters: _Incoming Requests_ and _Outgoing Requests_ filters.
```
Comment (danidoni) [inline _posts/development/2021-02-12-notifications-new-filters.md:25]:
```suggestion
Listening to the users' feedback, we have introduced a new pair of filters: _Incoming Requests_ and _Outgoing Requests_ filters.
```

File: _posts/development/2021-02-12-notifications-new-filters.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+_Incoming Requests_ displays all the unread notifications related to Requests that others created in packages where you are involved.
+_Outgoing Requests_ displays all the unread notifications related to requests which were created by yourself.
```
Comment (danidoni) [inline _posts/development/2021-02-12-notifications-new-filters.md:28]:
```suggestion
_Incoming Requests_ displays unread notifications related to Requests that others created in packages where you are involved.
_Outgoing Requests_ displays unread notifications related to requests which were created by yourself.
```

**Review by danidoni — APPROVED** [review]

File: _posts/development/2021-02-15-notifications-new-filters.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+Those of you enrolled in the [beta program](https://openbuildservice.org/2018/10/04/the-beta-program)
+already know the Notifications page we introduced a while ago.
+After a few iterations refining its functionality and UI, we recently added two more filters:
+_Incoming Requests_ and _Outgoing Requests_.
+
+## Filters Overview
+
+During a normal day, you may receive a big amount of web notifications. The notifications' filters are there
+to the rescue, by narrowing down the result' to the once you are looking for.
```
Comment (hennevogel) [inline _posts/development/2021-02-15-notifications-new-filters.md:15]:
```suggestion
to the rescue, by narrowing down the result' to the notifications you are looking for.
```

File: _posts/development/2021-02-15-notifications-new-filters.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+<figure>
+  <img src="/images/posts/notifications_incoming_outgoing_filters.png" alt="Notifications filter area" />
+  <figcaption>Notifications filter area</figcaption>
+</figure>
+
+In previous iterations we've already introduced different notifications' filters.
```
Comment (hennevogel) [inline _posts/development/2021-02-15-notifications-new-filters.md:22]: Please don't talk about things that like `iterations` that don't make sense to anyone but us :)

File: _posts/development/2021-02-15-notifications-new-filters.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+Listening to the users' feedback, we have created a new pair of filters: _Incoming Requests_ and _Outgoing Requests_ filters.
```
Comment (hennevogel) [inline _posts/development/2021-02-15-notifications-new-filters.md:23]:
```suggestion
Listening to the your feedback, we have created a new pair of filters: _Incoming Requests_ and _Outgoing Requests_ filters.
```

**Review by hennevogel — CHANGES_REQUESTED** [review]

**Comment by krauselukas** [comment]
@hennevogel please have another look

**Review by hennevogel — APPROVED** [review]

**Pull request merged**